engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/container/CallActivityProcessor.java

```diff
@@ -104,27 +104,27 @@ public void beforeExecutionPathCompleted(
       final ExecutableCallActivity element,
       final BpmnElementContext callActivityContext,
       final BpmnElementContext childContext) {
-    final var currentState = callActivityContext.getIntent();
-    switch (currentState) {
-      case ELEMENT_ACTIVATED:
-        // nothing to do any more
-        break;
-      case ELEMENT_TERMINATING:
-        // the call activity is already terminating, it doesn't matter that the child completed
-        // do nothing
-        break;
-      default:
-        final var message = String.format(UNABLE_TO_COMPLETE_FROM_STATE_MESSAGE, currentState);
-        throw new BpmnProcessingException(callActivityContext, message);
-    }
+    // nothing to do before the child is completed
```

Comment:
Should we add an empty body as default implementation of this `beforeExecutionPathCompleted` on the interface?